src/helpers/device.js

```diff
@@ -11,3 +11,5 @@ export const isIOS = window.navigator.userAgent.match(/(iPad|iPhone|iPod)/i) ||
 export const isAndroid = window.navigator.userAgent.match(/Android/i);
 export const isMobileDevice = isIOS || isAndroid || window.navigator.userAgent.match(/webOS|BlackBerry|IEMobile|Opera Mini/i);
 export const isMac = navigator.appVersion.indexOf('Mac') > -1;
+
+export const isChromeOniOS =  window.navigator.userAgent.match(/CriOS\/(.*?) /);
```

Comment:
One thing I'd like to discuss with you. Do you think it makes sense to use a browser detection library? For example this one https://www.npmjs.com/package/bowser

If we used this one we don't need to worry about updating this file or adding more browsers to it...

Reply:
Yea it seems like a good idea to this rather than doing it ourselves

Reply:
kk sounds good I will take a closer look at this package and talk to Matt later :)